src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -92,10 +106,17 @@ public void checkIndentation() {
         if (getLineStart(getMainAst()) == expandedTabsColumnNo(getMainAst())) {
             final IndentLevel level =
                 new IndentLevel(getIndent(), getIndentCheck().getLineWrappingIndentation());
-            if (!level.isAcceptable(expandedTabsColumnNo(getMainAst()))) {
+            if (isNonAcceptableIndent(getMainAst(), level)) {
+                isLambdaCorrectlyIndented = false;
                 logError(getMainAst(), "", expandedTabsColumnNo(getMainAst()), level);
             }
         }
     }
 
+    private boolean isNonAcceptableIndent(DetailAST firstChild, IndentLevel level) {
+        return expandedTabsColumnNo(firstChild) < level.getFirstIndentLevel()
+            || getIndentCheck().isForceStrictCondition()
+               && !level.isAcceptable(expandedTabsColumnNo(firstChild));
```

Comment:
Minor thing, since `expandedTabsColumnNo(firstChild)` is used twice, you could save the value to a variable.

Reply:
I have tried to the same but got unusual error: https://github.com/Abhishek-kumar09/checkstyle/pull/16/files

Reply:
I took a look at the previous changes. The exception isn't fully clear what it was, but I am going to assume you got an NPE because you were trying to use `firstChild` in areas where it was most likely `null`.
That can't be the case here, since you use it for both sides of the `||` condition.

Btw, now that I am looking at this a second time, should the first of the `||` have the following `!getIndentCheck().isForceStrictCondition()` ? Force strict should require that exact tab placement, and it can't be anything besides it.

Reply:
I think this logic is fine.
The Indentation is **NonAcceptable** if:
either the child is less than the minimum acceptable indent.
OR
strictCondition is true  and the child is not in the acceptable indent.

> should the first of the || have the following !getIndentCheck().isForceStrictCondition()

If the indent is less than minimum of acceptable indent, then doesnt matter if strict condition is true or false, It should  not be accepted...That's why I think this logic is not required.


> Force strict should require that exact tab placement, and it can't be anything besides it.

Currently It is following all the indent which is a part of accetable indent and nothing more or less that, only discrete acceptable values. I think it is still following the strictlyTrue condition.

Reply:
@rnveach 
Diff report with force strict condition true: https://abhishek-kumar09.github.io/github-actions-report-fetch/
Is it good, We can see here that there are more than 1 acceptable indents in some rare cases with true strict condition....
If it defeats the purpose of `trueForceStrictCondition` then I will change it.